### PR TITLE
Update release notes and changelog for 8.0

### PR DESCRIPTION
**Pull request by smgallo**
## Description

Updates prior to release.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: html/about/release_notes/xdmod.html [hunk 1]
```diff
@@ -20,10 +19,15 @@ <h2> Version 8.0.0 (2018-09-30) </h2>
                 </li>
                 <li>All XDMoD user profiles are now associated with an organization. Previously, this
                     was only required for Campus Champions.</li>
+                <li> Added support for automatically detecting / assigning a new SSO User's
+                    organization.</li>
+                <li> Added support for automatically detecting if a user's organization has changed
```
Comment (plessbd) [inline html/about/release_notes/xdmod.html]: Why does this item have a space after the opening `<li>` and the others dont?
```suggestion
                <li>Added support for automatically detecting if a user's organization has changed
```

File: html/about/release_notes/xdmod.html [hunk 2]
```diff
@@ -20,10 +19,15 @@ <h2> Version 8.0.0 (2018-09-30) </h2>
                 </li>
                 <li>All XDMoD user profiles are now associated with an organization. Previously, this
                     was only required for Campus Champions.</li>
+                <li> Added support for automatically detecting / assigning a new SSO User's
```
Comment (plessbd) [inline html/about/release_notes/xdmod.html]: Why does this item have a space after the opening `<li>` and the others dont?
```suggestion
                <li>Added support for automatically detecting / assigning a new SSO User's
```

**Comment by jtpalmer** [comment]
We've been maintaining a copy of the change log in the RPM spec file [open_xdmod/modules/xdmod/xdmod.spec.in](https://github.com/ubccr/xdmod/blob/xdmod8.0/open_xdmod/modules/xdmod/xdmod.spec.in#L90).  That at least needs a new release date as well.

**Comment by smgallo** [comment]
Updated spec file as per @jtpalmer. We really need to consolidate this into 1 place.

File: open_xdmod/modules/xdmod/xdmod.spec.in [hunk 3]
```diff
@@ -88,6 +88,40 @@ rm -rf $RPM_BUILD_ROOT
 %config(noreplace) %{_datadir}/%{name}/html/robots.txt
 
 %changelog
+* Tue Oct 29 2018 Steven M. Gallo <smgallo@buffalo.edu> 8.0.0-1.0
```
Comment (jtpalmer) [inline open_xdmod/modules/xdmod/xdmod.spec.in]:
```suggestion
* Tue Oct 30 2018 Steven M. Gallo <smgallo@buffalo.edu> 8.0.0-1.0
```

**Comment by smgallo** [comment]
Addressed suggestion from @jtpalmer

**Pull request merged**